lib/parseMarkdown.js

```diff
@@ -12,33 +12,112 @@ const {
   whole,
   wildcard,
   extra,
+  capture,
   and,
-  matchers: { WHITE_SPACE: SPACE, START, LAZY },
+  or,
+  matchers: { ANY, WHITE_SPACE: SPACE, START, LAZY },
   flags
 } = require('rexrex')
 
 const space = extra(SPACE)
 const isCommandReTemplate = and(
   START, // ^
   wildcard(SPACE),
-  ['Runs' + LAZY, 'tasks' + LAZY].join(space)
+  [
+    'Runs' + LAZY,
+    'tasks' + LAZY,
+    capture(extra(ANY)),
+    capture(or('after', 'before')),
+    'this'
+  ].join(space),
+  capture(
+    or(
+      and(space, 'and', space, 'watch', space, capture(extra(ANY))),
+      and(space, 'in', space, 'parallel')
+    )
+  ) + LAZY,
+  capture(and(space, 'and', space, 'watch', space, capture(extra(ANY)))) + LAZY,
+  wildcard(SPACE)
```

Comment:
Actually, this now enables several expressions:
```
Run task `lint` in parallel and watch ...
```
```
Run tasks `lint`, `build` and `clean` in parallel and watch ...
```
```
Run task `lint` after this / before this in parallel and watch ...
```
```
Run task `lint` and watch ...
```

Reply:
And I really don't think the user feels more convinient writing 
```
Run task `lint` after in parallel
```
instead of
```
Run task `lint` after this in parallel
```
as latter would be a proper sentence